src/components/PunsForm/PunsForm.css

```diff
@@ -0,0 +1,47 @@
+.form {
+  width: 80vw;
+  display: grid;
```

Comment:
why display:grid here?
i didnt see/understand why grid and not flex, i saw `gap` in the code, that u can use with flex but i didnt see any other grid use like `column-gap row-gap` or any other grid property.
Theres an actual change in the css? Can u enlighten me ,please?

Reply:
No, I was having trouble to set the 'justify -self' in the button and the devtools suggested set the display grid... I did just to achieve the positioning at the time and forgot to change!